controllers/teams.js

```diff
@@ -1052,6 +1052,48 @@ router.get('/:teamId/members', async (req, res, next) => {
 			}
 		}
 
+		let files;
+
+		files = await api(req).get('/fileStorage', {
+			qs: {
+				owner: req.params.teamId,
+			},
+		});
+
+		/* note: fileStorage can return arrays and error objects */
+		if (!Array.isArray(files)) {
+			if ((files || {}).code) {
+				logger.warn(files);
+			}
+			files = [];
+		}
+
+		files = files.filter((file) => file);
+		files = files.map((file) => {
+			file.saveName = file.name.replace(/'/g, "\\'");
+			if (file && file.permissions) {
+				file.permissions = mapPermissionRoles(file.permissions, roles);
+				return file;
+			}
+			return undefined;
+		});
+
+		files = files.filter((f) => !f.isDirectory);
+
+		files
+			.sort((a, b) => {
+				if (b && b.updatedAt && a && a.updatedAt) {
+					return timesHelper.fromUTC(b.updatedAt) - timesHelper.fromUTC(a.updatedAt);
+				}
+				return 0;
+			});
+
+		team.userIds.map((user) => {
```

Comment:
map always returns value, that we don't use here, use forEach for cases where we don't need transformed array.